.gitignore

```diff
@@ -0,0 +1,4 @@
+.vscode/
+node_modules
+package-lock.json
+package.json
```

Comment:
- [ ] Kindly remove   `package.json`   from the `gitignore`  file.  `package.json` file is a critical file that contains important information about your project's dependencies and configuration. If you exclude package.json from version control, other developers who clone your repository will not be able to install the required dependencies to run the project, as they won't know what dependencies are required. Also note that in your readme file, you included the `installation` and `running` commands that require a `package.json` file